test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml

```diff
@@ -0,0 +1,60 @@
+name: "top level field is removed for a subscription with namespaced field"
+enabled: true
+# language=GraphQL
+overallSchema:
+  CommentService: |
+    directive @toBeDeleted on FIELD_DEFINITION
+    type Query {
+      commentById(id: ID): Comment @toBeDeleted
+    }
+    type Subscription {
+      commentsApi: CommentsApi @namespaced
+    }
+    type CommentsApi {
+      onCommentUpdated(id: ID): Comment @toBeDeleted
+    }
+    type Comment {
+      id: ID
+    }
+# language=GraphQL
+underlyingSchema:
+  CommentService: |
+    type Query {
+      commentById(id: ID): Comment
+    }
+    type Subscription {
+      commentsApi: CommentsApi
+    }
+    type CommentsApi {
+      onCommentUpdated(id: ID): Comment
+    }
+    type Comment {
+      id: ID
+    }
+# language=GraphQL
+query: |
+  subscription {
+    commentsApi {
+      onCommentUpdated(id: "C1") {
+        id
+      }
+    }
+  }
+variables: { }
+serviceCalls: [ ]
+# language=JSON
+response: |-
+  {
+    "errors": [
+      {
+        "locations": [],
+        "message": "null",
+        "extensions": {
+          "classification": "ValidationError"
```

Comment:
this error is coming from a "remove field" transform that we use for tests. I activated this transform with `onCommentUpdated(id: ID): Comment @toBeDeleted` directive in the overall schema

Reply:
previously we would blow up with `your data fetcher must return a publisher ...` kind of an error